content/examples/shortcodes/ref-relref/en.md

```diff
@@ -23,3 +24,20 @@ The result will be just like a link. But the difference is that with ref/relref
 
 [Buttons example]({{< relref "buttons/index.md" >}})
 
+Relref can also be used for anchor links:
```

Comment:
If I'm interpretting this change correctly, please add a note that same-page links to a fragment "*must*" be invoked via `ref`/`relref`, if they're to work consistently.